src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.ToolStripAccessibleObject.cs

```diff
@@ -206,6 +222,64 @@ internal int GetChildOverflowFragmentCount()
                 return _owningToolStrip.OverflowItems.Count;
             }
 
+            /// <summary>
+            ///  Returns the next or the previous ToolStrip item, that is not a ToolStripControlHost
+            ///  or is a control, that supports UIA providers. In fact, using this method removes hosted ToolStrip items,
+            ///  that are native controls (doesn't support UIA providers, their accessible objects is Windows responsability),
+            ///  from the accessibility tree. It's necessary, because hosted native controls internally add accessible objects
+            ///  to the accessibility tree. To avoid duplicates, remove hosted items with native accessibility objects from the tree.
+            /// </summary>
```

Comment:
Few issues with this comment.

1. "_... that is not a ToolStripControlHost or is a control, that supports UIA providers..._" - statements like "this is not A or is B" are very hard to parse.
2. Spelling errors

```suggestion
            /// <summary>
            ///  Returns the next or the previous ToolStrip item, that is considered a valid navigation fragment (e.g. a control,
            ///  that supports UIA providers and not a <see cref="ToolStripControlHost"/>).
            ///  This method removes hosted ToolStrip items that are native controls (their accessible objects are provided by Windows)
            ///  from the accessibility tree. It's necessary, because hosted native controls add accessible objects
            ///  to the accessibility tree, and thus create duplicated. To avoid duplicates, remove hosted items with native accessibility objects from the tree.
            /// </summary>
```

Please reformat as appropriate.